src/corpus/ui/ChapterViewLine.tsx

```diff
@@ -286,7 +287,14 @@ export function ChapterViewLineVariant({
   )
 
   return (
-    <>
+    <LineInfoContext.Provider
```

Comment:
The popover needs `chapter.id`, `lineNumber`, `variantNumber` and `textService` to fetch corresponding manuscript lines. As the popover is deeply nested and this information should also be shared by `Score` (which is located elsewhere in the tree), it seems reasonable to create it as context.